ote/test/clj/ote/transit_changes/detection_integration_test.clj

```diff
@@ -0,0 +1,148 @@
+(ns ote.transit-changes.detection-integration-test
+  (:require [ote.transit-changes.detection :as detection]
+            [ote.services.transport :as transport-service]
+            [ote.db.transport-service :as t-service]
+            [specql.core :as specql]
+            [ote.integration.import.gtfs :as gtfs-import]
+            [clojure.test :as t :refer [deftest testing is use-fixtures]]
+            [clojure.spec.test.alpha :as spec-test]
+            [ote.transit-changes :as transit-changes]
+            [ote.transit-changes.detection-test-utilities :as tu]
+            [ote.time :as time]
+            [com.stuartsierra.component :as component]
+            [ote.test]))
+
+(defn slurp-bytes
```

Comment:
No need to do anything if you don't want to, but just wondering if this should be moved to some utility file?